tests/tecxc

```diff
@@ -2,9 +2,9 @@
 # Copyright (C) 2023 Simo Sorce <simo@redhat.com>
 # SPDX-License-Identifier: Apache-2.0
 
-# On Fedora/EL completely removed support for explicit EC from libcrypto,
-# so skip the test completely
-if [ -f /etc/redhat-release ]; then
+# Some distributions completely removed support for explicit EC from libcrypto.
+# If `-Denable_explicit_EC_test=true` is not set, skip the test.
+if [ -z "${ENABLE_EXPLICIT_EC_TEST}" ]; then
     exit 0
```

Comment:
I see this is not a change you made, but I would prefer to have here `exit 77` to be clear that the test was not executed and was skipped similarly as with the other tests that were skipped.